### ISSUE TITLE
Can't backup if WSA is not installed in the main OS disk

### ISSUE THREAD

**Issue by ghost** [Simizfo/WSATools#42]
Hello！It's really a nice tool for us.😙❤️

I encountered a problem when I first used it.

I don't like the computer's C disk is too huge.So I moved the wsa to disk D through the computer system settings.When I backup, the error path is still in disk C.(or Is it because the software does not have administrator rights？)

- _My solution is:Move the main program to disk C again.Final success.__

**Comment by Simizfo** [comment]
Hi!

First of all, thanks for reporting.
This, uhhh... shouldn't be possible? WSA is an app that cannot be installed on other than the main disk.

Maybe you installed it manually? 

But well, yeah. WSATools looks for WSA to be in the main disk. Maybe I should make this more flexible...

**Comment by Simizfo** [comment]
Yes, I know about this, but if I remember correctly, the WSA app manifest says that it must stay on the main disk. Weird.

Well anyway, it's something I can fix on my end, so I'll do that!

**Comment by LittleFreak** [comment]
Not sure, if its the same problem, but my WSATools freezes instantly, once I try to create a backup. Browsing for existent Backups are working though. Interesting enough is, that the freeze also appears, when I try to change the experimental "shared path" folder within the main WSA settings. idk if that's related either.
Log output:
![image](https://github.com/Simizfo/WSATools/assets/53310735/af750588-e519-40e7-8b29-3673449e31b8)
